### ISSUE TITLE
how to close log 

### ISSUE THREAD

**Issue by Ericliuzi** [scrapy/scrapyd#298]
how to close the log like """2018-10-22T14:05:34+0800 [twisted.python.log#info] "127.0.0.1" - - [22/Oct/2018:06:05:33 +0000] "GET /listjobs.json?project=test HTTP/1.1" 200 64 "-" "python-requests/2.13.0" """ 

I check the code ,but i do not find where can close this log .
can you help me ?

**Comment by Digenis** [comment]
@Ericliuzi I don't understand your question.
Can you rephrase it?

**Comment by my8100** [comment]
It seems that there is an application using the requests library to call the *listjobs.json* API from 127.0.0.1
